packages/admin/cms-admin/src/warnings/WarningsGrid.tsx

```diff
@@ -115,11 +122,30 @@ export function WarningsGrid() {
             width: 150,
             renderCell: (params) => <WarningSeverity severity={params.value} />,
         },
+        {
+            field: "nameInfo",
+            headerName: intl.formatMessage({ id: "warning.nameAndInfo", defaultMessage: "Name/Info" }),
+            sortable: false,
+            filterable: false,
+            width: 200,
+            renderCell: ({ row }) => {
+                return (
+                    <GridCellContent
+                        primaryText={row.entityInfo?.name ?? <FormattedMessage {...messages.unknown} />}
+                        secondaryText={row.entityInfo?.secondaryInformation}
+                    />
+                );
+            },
+        },
         {
             field: "type",
             headerName: intl.formatMessage({ id: "warning.type", defaultMessage: "Type" }),
-            width: 150,
-            renderCell: (params) => <Chip label={params.value} />,
+            sortable: false,
+            filterable: false,
+            width: 100,
+            renderCell: ({ row }) => (
+                <Chip label={entityDependencyMap[row.sourceInfo.rootEntityName]?.displayName ?? row.sourceInfo.rootEntityName} />
+            ),
```

Comment:
Now that we use the entity info instead of the type, what do we use the warning type for?

Reply:
I wasn't sure so I tested it, but it works without type, yes: https://github.com/vivid-planet/comet/pull/3904